database/migrations/2018_02_06_133404_add_is_admin_user.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddIsAdminUser extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+   
+	public function up()
+	{
+	    Schema::table('users', function (Blueprint $table) {
```

Comment:
no need for is_admin , just admin - bool.

Reply:
done